CMake/SofaPython3Tools.cmake

```diff
@@ -201,34 +202,39 @@ function(SP3_add_python_module)
     #    We compute its path relative to this target output file
     #    Ex: ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/site-packages/Sofa  --> $ORIGIN/../Sofa
     # 3. Add the relative path computed in 2 to the list of RPATHS
-    get_target_property(${A_TARGET}_DEPENDENCIES_RPATH ${A_TARGET} "INSTALL_RPATH")
-    foreach(DEPENDENCY ${DEPENDS_ALL})
-        if(NOT TARGET ${DEPENDENCY})
-            continue()
-        endif()
-        get_target_property(aliased_dep ${DEPENDENCY} ALIASED_TARGET)
-        if(aliased_dep)
-            set(DEPENDENCY ${aliased_dep})
-        endif()
-        get_target_property(DEPENDENCY_LIBRARY_OUTPUT_DIRECTORY "${DEPENDENCY}" LIBRARY_OUTPUT_DIRECTORY)
-        if (DEPENDENCY_LIBRARY_OUTPUT_DIRECTORY)
-            file(RELATIVE_PATH dependency_path_from_packages "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${SP3_PYTHON_PACKAGES_DIRECTORY}" "${DEPENDENCY_LIBRARY_OUTPUT_DIRECTORY}")
-            if (NOT "${dependency_path_from_packages}" STREQUAL "" AND NOT "${dependency_path_from_packages}" STREQUAL "../")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "$ORIGIN/../${dependency_path_from_packages}")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "$$ORIGIN/../${dependency_path_from_packages}")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "@loader_path/../${dependency_path_from_packages}")
-                list(APPEND ${A_TARGET}_DEPENDENCIES_RPATH "@executable_path/../${dependency_path_from_packages}")
-            endif()
-        endif()
-    endforeach()
+    get_target_property(TARGET_DEPENDENCIES_RPATH ${A_TARGET} "INSTALL_RPATH")
+
```

Comment:
You removed the loop on the SP3 dependencies? Won't this break when, for example, 
```
lib/python3/site-packages/SofaTypes/SofaTypes.so
```
 depends on 
```
lib/python3/site-packages/Sofa/Core.so
```

Reply:
Ok that was the purpose of this loop. Thanks, I will put it back :laughing:

Reply:
Done !